extensions/spring-cloud-config-client/runtime/src/main/java/io/quarkus/spring/cloud/config/client/runtime/SpringCloudConfigClientConfig.java

```diff
@@ -96,7 +97,13 @@
      * When using HTTPS and no keyStore has been specified, whether or not to trust all certificates
      */
     @ConfigItem(defaultValue = "false")
-    boolean trustCerts;
+    public boolean trustCerts;
+
+    /**
+     * Custom headers to pass the Spring Cloud Config Server when performing the HTTP request
+     */
+    @ConfigItem
+    public Map<String, String> headers;
```

Comment:
The HTTP spec [RFC 2616][1] allows multiple values for a single HTTP header: 

> Multiple message-header fields with the same field-name MAY be present in a message if and only if the entire field-value for that header field is defined as a comma-separated list [i.e., #(values)]. It MUST be possible to combine the multiple header fields into one "field-name: field-value" pair, without changing the semantics of the message, by appending each subsequent field-value to the first, each separated by a comma. The order in which header fields with the same field-name are received is therefore significant to the interpretation of the combined field value, and thus a proxy MUST NOT change the order of these field values when a message is forwarded.

What this means is that you can send multiple values in a single header by concatenating them with commas. Do you want to support that here or that wouldn't matter?

[1]: http://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2

Reply:
Yeah I know, but I really don't think it's necessary. Best keep the common case easy to use.